packages/make-styles/src/runtime/resolveStyleRules.ts

```diff
@@ -28,9 +51,10 @@ export function resolveStyleRules(
   pseudo = '',
   media = '',
   support = '',
-  result: Record<string, MakeStylesResolvedRule> = {},
```

Comment:
`resolveStyleRules` accepts 8 arguments, 6 of them are used only internally when called recursively and are not supposed to be passed from the outside.
Have you considered splitting the function to the public interface and the internal function?

Reply:
Yeah, it's a good and https://github.com/microsoft/fluentui/pull/18169#discussion_r659579766 has a clear motivation to do it, thanks for the suggestion 👍

Reply:
Done in 3c6c50a.